roles/softserve/templates/application.cfg

```diff
@@ -0,0 +1,7 @@
+DEBUG = False
+SQLALCHEMY_DATABASE_URI = 'postgresql://{{ softserve_config['db_user'] }}:{{ softserve_config['db_pass'] }}@{{ softserve_config['db_host'] }}/{{ softserve_config['db_name'] }}
+SECRET_KEY = {{ softserve_config['app_secret_key'] }}
+AUTH_SYSTEM = 'rackspace'
+AUTH_SYSTEM_REGION = 'ORD'
+USERNAME = {{ softserve_config['rackspace_username'] }}
```

Comment:
Here too, it would surely be safer to add quote around the value. And the 2 keys as well.